yarn.lock

```diff
@@ -1880,6 +1894,11 @@ detect-newline@^2.1.0:
   resolved "https://registry.yarnpkg.com/detect-newline/-/detect-newline-2.1.0.tgz#f41f1c10be4b00e87b5f13da680759f2c5bfd3e2"
   integrity sha1-9B8cEL5LAOh7XxPaaAdZ8sW/0+I=
 
+diagnostic-channel-publishers@0.4.3:
+  version "0.4.3"
+  resolved "https://registry.yarnpkg.com/diagnostic-channel-publishers/-/diagnostic-channel-publishers-0.4.3.tgz#b177fdf3bfc325fbc69e39e1cccfe6656cf2fca9"
+  integrity sha512-E3Fyg41SJd2GbLC63fkAaqsQRLVMKptpnZ0HoDsRYmqOVd92HLIt/c/EZqYnANM9+YPU3H1lx+GmjAMZWs65Nw==
+
 diagnostic-channel-publishers@^0.3.3:
   version "0.3.3"
   resolved "https://registry.yarnpkg.com/diagnostic-channel-publishers/-/diagnostic-channel-publishers-0.3.3.tgz#376b7798f4fa90f37eb4f94d2caca611b0e9c330"
```

Comment:
Maybe we have to relock?

Reply:
```sh
yarn why diagnostic-channel
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "diagnostic-channel"...?
[2/4] 🚚  Initialising dependency graph...
warning Resolution field "fp-ts@1.17.4" is incompatible with requested version "fp-ts@1.12.0"
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "diagnostic-channel@0.2.0"
info Reasons this module exists
   - "applicationinsights" depends on it
   - Hoisted from "applicationinsights#diagnostic-channel"
info Disk size without dependencies: "144KB"
info Disk size with unique dependencies: "228KB"
info Disk size with transitive dependencies: "228KB"
info Number of shared dependencies: 1
=> Found "@pagopa/ts-commons#diagnostic-channel@0.3.1"
info Reasons this module exists
   - "@pagopa#ts-commons#applicationinsights" depends on it
   - Hoisted from "@pagopa#ts-commons#applicationinsights#diagnostic-channel"
info Disk size without dependencies: "56KB"
info Disk size with unique dependencies: "140KB"
info Disk size with transitive dependencies: "140KB"
info Number of shared dependencies: 1
```

Reply:
```sh
yarn why diagnostic-channel-publishers
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "diagnostic-channel-publishers"...?
[2/4] 🚚  Initialising dependency graph...
warning Resolution field "fp-ts@1.17.4" is incompatible with requested version "fp-ts@1.12.0"
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "diagnostic-channel-publishers@0.3.3"
info Reasons this module exists
   - "applicationinsights" depends on it
   - Hoisted from "applicationinsights#diagnostic-channel-publishers"
info Disk size without dependencies: "116KB"
info Disk size with unique dependencies: "116KB"
info Disk size with transitive dependencies: "116KB"
info Number of shared dependencies: 0
=> Found "@pagopa/ts-commons#diagnostic-channel-publishers@0.4.3"
info Reasons this module exists
   - "@pagopa#ts-commons#applicationinsights" depends on it
   - Hoisted from "@pagopa#ts-commons#applicationinsights#diagnostic-channel-publishers"
info Disk size without dependencies: "140KB"
info Disk size with unique dependencies: "140KB"
info Disk size with transitive dependencies: "140KB"
info Number of shared dependencies: 0
```

Reply:
@AleDore I didn't get the question. Do you there's something we should do?

Reply:
The question is: Why we have 2 different version of `diagnostic-channel-publishers",` can we resolve only one version?

Reply:
`diagnostic-channel-publishers` is a dep of `applicationinsights`, so the same applies to the latter:
```
yarn why applicationinsights
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "applicationinsights"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "applicationinsights@1.7.3"
info Has been hoisted to "applicationinsights"
info This module exists because it's specified in "dependencies".
info Disk size without dependencies: "1.63MB"
info Disk size with unique dependencies: "2.21MB"
info Disk size with transitive dependencies: "3.11MB"
info Number of shared dependencies: 5
=> Found "@pagopa/ts-commons#applicationinsights@1.8.9"
info This module exists because "@pagopa#ts-commons" depends on it.
info Disk size without dependencies: "1.86MB"
info Disk size with unique dependencies: "2.45MB"
info Disk size with transitive dependencies: "3.34MB"
info Number of shared dependencies: 5
```
The point is that dependency to `applicationinsights` is declared both here and inside `@pagopa/ts-commons`, so yarn resolves both dependendency independently. 

I don't know if there's a case for which this is a problem, I can't see any so far. Do you?

Reply:
Ok, let's approve 👍